File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md

```diff
@@ -452,6 +452,45 @@ The runtime files, or blink binaries, will be copied into a bin or published fol
 
 By including the <b><ExcludeAssets>native</ExcludeAssets></b> option in the package reference of the csproj file, you can exclude the runtime files or blink binaries from being copied into the bin or publish folder while building and publishing the application. But you need to place the BlinkBinaries in the server disk and set the BlinkPath in the BlinkConverterSettings to perform the conversion. 
 
+## Preservation issues occurs while converting HTML(HTTPS sites) to PDF using WebKit rendering engine in local machine/Azure App service
```

Comment:
Use this title "Image rendering issue occurs while converting Html to PDF using WebKit"